assets/src/scripts/editoria11y-tinymce.js

```diff
@@ -0,0 +1,298 @@
+		base.customTests = base.customTests || 0;
+		base.preventCheckingIfPresent = '';
+		return base;
+	}
+
+	/**
+	 * Ensure Ed11y script is loaded inside iframe and constructor exported.
+	 *
+	 * @param {Window} iframeWin
+	 * @param {Function} done
+	 */
+	function ensureEd11yInIframe( iframeWin, done ) {
+		if ( iframeWin.Ed11y ) {
+			return done();
+		}
+		const parentScript = document.querySelector( 'script[src*="editoria11y.min.js"]' );
+		if ( ! parentScript ) {
+			return done();
+		}
+		if ( iframeWin.document.querySelector( 'script[src*="editoria11y.min.js"]' ) ) {
+			let loops = 0;
+			( function waitCtor() {
+				try {
+					if ( ! iframeWin.Ed11y && ! iframeWin.__pbEd11yExportTried ) {
+						iframeWin.__pbEd11yExportTried = true;
+						const exp = iframeWin.document.createElement( 'script' );
+						exp.text = EXPORT_SHIM;
+						iframeWin.document.head.appendChild( exp );
+					} else if ( ! iframeWin.Ed11y && iframeWin.__pbEd11yExportTried ) {
+						const exp2 = iframeWin.document.createElement( 'script' );
+						exp2.text = EXPORT_SHIM;
+						iframeWin.document.head.appendChild( exp2 );
+					}
+				} catch ( e ) { /* capture constructor polling error */ }
+				if ( iframeWin.Ed11y ) {
+					return done();
+				}
+				if ( window.Ed11y && ! iframeWin.Ed11y ) {
+					iframeWin.Ed11y = window.Ed11y;
+					return done();
+				}
+				if ( loops++ > 40 ) {
+					return done();
+				}
+				setTimeout( waitCtor, POLL_INTERVAL );
+			} )();
+			return;
+		}
+		const s = iframeWin.document.createElement( 'script' );
+		s.src = parentScript.src;
+		/**
+		 *
+		 */
+		s.onload = function () {
+			let loops2 = 0;
+			( function confirmCtor() {
+				try {
+					if ( ! iframeWin.Ed11y ) {
+						const exp3 = iframeWin.document.createElement( 'script' );
+						exp3.text = EXPORT_SHIM;
+						iframeWin.document.head.appendChild( exp3 );
+					}
+				} catch ( e2 ) { /* ignore post-load export error */ }
+				if ( iframeWin.Ed11y ) {
+					return done();
+				}
+				if ( window.Ed11y && ! iframeWin.Ed11y ) {
+					iframeWin.Ed11y = window.Ed11y;
+					return done();
+				}
+				if ( loops2++ > 25 ) {
+					return done();
+				}
+				setTimeout( confirmCtor, 120 );
+			} )();
+		};
+		/**
+		 *
+		 */
+		s.onerror = function () {
+			done();
+		};
+		iframeWin.document.head.appendChild( s );
+	}
+
+	/**
+	 * Ensure stylesheet is present in iframe.
+	 *
+	 * @param {Document} iframeDoc
+	 */
+	function ensureIframeStyles( iframeDoc ) {
+		const parentCss = document.querySelector( 'link[href*="editoria11y"][href$=".css"], link[href*="editoria11y.min.css"]' );
+		if ( parentCss && ! iframeDoc.querySelector( 'link[href="' + parentCss.href + '"]' ) ) {
+			const l = iframeDoc.createElement( 'link' );
+			l.rel = 'stylesheet';
+			l.href = parentCss.href;
+			iframeDoc.head.appendChild( l );
+		}
+	}
+
+	/**
+	 * Create Ed11y instance for an editor iframe.
+	 *
+	 * @param {object} editor
+	 */
+	function initEd11yInstance( editor ) {
```

Comment:
wait for TinyMCE editors, then for iframe body, then for the Ed11y constructor with bounded retry loops and timeouts